radixdlt/src/integration/java/test/integration/AttackTests.java

```diff
@@ -24,127 +24,123 @@
 import java.util.Map;
 import java.util.Random;
 
-public class AttackTests
-{
+public class AttackTests {

```

Comment:
Do we still need this? It doesn't seem relevant anymore, e.g. gossip discovery tables are irrelevant now.